app/views/application_letters/check.html.erb

```diff
@@ -1,16 +1,22 @@
 <%- model_class = ApplicationLetter -%>
 
+<p class='alert alert-info' >
+  <% if @application_deadline_exceeded %>
+    <%= t('.deadline_exceeded') %>
+  <% else %>
+    <%= t('.can_change_until', application_deadline: l(@application_letter.event.application_deadline)) %>
+  <% end %>
+</p>
 <div class="page-header">
   <h1><%=t '.check_application_for', event_name: @application_letter.event.name %></h1>
-  <p class='alert alert-info' >
-    <% if @application_deadline_exceeded %>
-      <%= t('.deadline_exceeded') %>
-    <% else %>
-      <%= t('.can_change_until', application_deadline: l(@application_letter.event.application_deadline)) %>
-    <% end %>
-  </p>
 </div>
 
+<%# Be careful: user.requires_agreement_letter_for_event? behaves exactly the way you wouldn't expect it do behave: If the user requires an agreement letter for the event, it returns false... But fuck me if I'm the one to refactor it%>
```

Comment:
Yea, same here, but unfortunately the scope of the user story did not allow any further work invested into it